doc/application-examples.md

```diff
@@ -6,3 +6,8 @@
 
 ## MD on different hardware platforms
 
+There is a scheme to generate doc html from python scripts provided in the ../examples/ directory, so that each case in doc has a counter part in ../examples.
+
+This scheme is implemented with sphinx-gallery.
+
+Do we want to use this ???
```

Comment:
Why do we need this file? what does the text mean ?

Reply:
This is about toy model. If we are going to add something like "application examples", there is a scheme called sphinx-gallery, where we put toy models in example file, and generate demos automatically to display in doc.

Reply:
We don't have toy model yet. So this is just a suggestion to put in bank.

Reply:
see examples/* and doc/train-*.md for latest updates on the examples.